--- a/dist/changesets-cli.cjs.js
+++ b/dist/changesets-cli.cjs.js
@@ -877,7 +877,7 @@
 // we check `npm info` before publishing but `npm info` can return stale data at times
 // so we need to gracefully handle this situation
 function isAlreadyPublishedError(output) {
-  return output.includes("cannot publish over the previously published version");
+  return typeof output === "string" && output.includes("cannot publish over the previously published version");
 }
 // we have this so that we can do try a publish again after a publish without
 // the call being wrapped in the npm request limit and causing the publishes to potentially never run
